src/gsi/m_berror_stats.f90

```diff
@@ -11,6 +11,7 @@ module m_berror_stats
 ! program history log:
 !   2010-03-24  j guo   - added this document block
 !   2011-08-01  lueken  - changed F90 to f90 (no machine logic) and fix indentation
+!   2014-04-01  weir    - added some chem support
```

Comment:
Why is this change from 2014 appearing?

Reply:
@CoryMartin-NOAA asked a[ similar question](https://github.com/NOAA-EMC/GSI/pull/810#discussion_r1866029539).  

Routine `setcorchem_` is in both `develop` ([here](https://github.com/NOAA-EMC/GSI/blob/9ada88e8b080c37a5146092e6a0a8dfc45b6654b/src/gsi/m_berror_stats.f90#L783)) and this PR ([here](https://github.com/RussTreadon-NOAA/GSI/blob/0e07d71fdd846c5590e44285f14ad4b9f23f94c9/src/gsi/m_berror_stats.f90#L1002)).  The comment above is only in this PR.  My guess is that the GSI code which Ricardo started from differs from EMC's code.

This is not an argument to retain the chem code ... just noting that this PR did not add the code.